content/api/commands/attachfile.md

```diff
@@ -0,0 +1,213 @@
+---
```

Comment:
Be sure to add the new page to https://github.com/cypress-io/cypress-documentation/blob/master/content/_data/sidebar.json so it gets a spot in the list & API TOC.

Reply:
You'll also need to add the new page to `content/_data/en.json`

Does `attachFile` also follow Actionability logic? If so we'll want to add it as a file that follows the logic to the `content/guides/core-concepts/interacting-with-elements.md` page.

Reply:
Added in all three places.